app/src/test/java/com/sefford/artdrian/datasources/WallpaperRepositoryTest.kt

```diff
@@ -66,7 +67,7 @@ class WallpaperRepositoryTest {
             val local = WallpaperMemoryDataSource()
             val repository = WallpaperRepository(api, local)
 
-            local.saveMetadata(listOf(SECOND_METADATA))
+            local.saveMetadata(listOf(SECOND_METADATADTO).map { it.toDomain() })
```

Comment:
We don't charge for the underscore (SECOND_METADATA_DTO)

Reply:
I'll change it